### PR TITLE
Incremental changes to verrazzano system roles

### PR DESCRIPTION
**Pull request by wmhopkins**
# Description

Update verrazzano user-facing system roles:
- don't give write access to */status resources of install.verrazzano.io
- add clusters.verrazzano.io to admin and monitor roles
- add istio read permission to admin and monitor roles
- update tests

This branch is rooted in mark/vz-2077, which adds tests for the user-facing roles (PR: https://github.com/verrazzano/verrazzano/pull/607). That branch won't need to be merged if this branch is merged first.

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [x] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/pkg/kubernetes.go [hunk 1]
```diff
@@ -339,6 +340,58 @@ func DeleteNamespace(name string) error {
 	return err
 }
 
+// DoesClusterRoleExist returns whether a cluster role with the given name exists in the cluster
+func DoesClusterRoleExist(name string) bool {
+	// Get the kubernetes clientset
```
Comment (markxnelson) [inline tests/e2e/pkg/kubernetes.go:345]: Kubernetes has capital letter - applies to several lines

**Review by markxnelson — DISMISSED** [review]
approved, but there are several places where spelling needs updating

File: tests/e2e/pkg/kubernetes.go (hunk 1 above)
Comment (wmhopkins) [reply tests/e2e/pkg/kubernetes.go]: Updated kubernetes to Kubernetes in all comments.

**Comment by wmhopkins** [comment]
> approved, but there are several places where spelling needs updating

I saw and fixed the kubernetes capitalization; was that it? Looks like you'll need to re-approve due to the update.

**Comment by wmhopkins** [comment]
Added one more change: rbac import renamed from v1 to rbacv1.

**Review by markxnelson — APPROVED** [review]

**Pull request merged**